keyboards/dz60/keymaps/joooosh/keymap.c

```diff
@@ -0,0 +1,50 @@
+#include QMK_KEYBOARD_H
+
+#define ______ KC_TRNS
```

Comment:
QMK defines a readability-oriented alias for this already.

https://github.com/qmk/qmk_firmware/blob/60b2a9a5ea88321629e970e936652cc1ba786b80/quantum/quantum_keycodes.h#L29-L31

If you use that one in your keymap you don't need to define this alias.

Reply:
I've updated keymap.c accordingly.